src/client/app/components/LineChartComponent.tsx

```diff
@@ -86,8 +95,17 @@ export default function LineChartComponent() {
 				}}
 				config={{
 					responsive: true,
-					displayModeBar: false,
-					// Current Locale
+					displayModeBar: true,
+					modeBarButtonsToRemove: listOfButtons,
+					modeBarButtonsToAdd: [{
+						name: 'more-options',
+						title: 'More Options',
```

Comment:
This comment applies across all the components changed. Two things:
1. Could this be internationalized with translate() and data.ts.
2. I'm thinking making the text 'Toggle Options' might be better since it is the same text whether you are going to all or few icons. What do you think?